specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,54 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
+        "summary": "Gets subscription core usages for location, for a given type of quota.",
+        "description": "Gets the usage in cores for a subscription in a given location, for a specific quota type.",
+        "operationId": "GetUsagesInLocation",
```

Comment:
should be end with _list:

https://github.com/Azure/azure-openapi-validator/blob/main/docs/list-in-operation-name.md